### DISCUSSION TITLE
Default / initial guess

### DISCUSSION THREAD

**Discussion by TinusUU** [mwshinn/PyDDM#113] [Ask for help] [answered]
Hi, 

In the object-oriented version, one could add an initial guess for each parameter, in addition to the fit-limits (default value).
(e.g. Fittable(minval=10, maxval=100, default_value=20) 
Is this still possible with the 0.8.1 update?  
I tried adding a third number to parameter limits, but that didn't work ;-)

Best,
Martijn

**Comment by mwshinn** [comment, marked as answer]
In the gddm function interface, instead of the tuple, you can just pass a Fittable object in the parameters dictionary.
